lib/setup-win32.h

```diff
@@ -67,10 +63,6 @@
 
 #ifdef HAVE_WINSOCK2_H
 #  define USE_WINSOCK 2
-#else
-#  ifdef HAVE_WINSOCK_H
-#    error "WinSock version 1 is no longer supported, version 2 is required!"
```

Comment:
Doesn't it make sense to keep this error condition here?

Reply:
That would mean keeping the variable, and keeping the checks for winsock.h in the build systems. Do you think that's worth it for the case that winsock.h is available but winsock2.h isn't? I'd just treat that exactly the same as when neither is available for simplicity.

Reply:
Fine with me, thanks for explaining it.